js/Member.js

```diff
@@ -0,0 +1,59 @@
+import Component from "./core/Component.js";
+import { memberAPI, teamAPI } from "./api/api.js";
+import TodoApp from "./components/TodoApp.js";
+import MemberAppender from "./components/MemberAppender.js";
+class Member extends Component {
+  async setup() {
+    const teamID = location.hash.split("#")[1];
+    const team = await teamAPI.getTeam(teamID);
+    const members = team.members;
+    this.state = {
+      teamID,
+      team,
+      members,
+    };
+  }
+  template() {
+    const { team } = this.state;
+    return `
+      <h1 id="user-title" data-username="eastjun">
+        <span><strong>${team.name}</strong>'s Todo List</span>
+      </h1>
+      <ul class="todoapp-list-container flex-column-container">
+      <li class="add-user-button-container">
+        
+      </li>
+      </ul>
+    `;
```

Comment:
```suggestion
      <h1 id="user-title" data-username="eastjun">
        <span><strong>${team.name}</strong>'s Todo List</span>
      </h1>
      <ul class="todoapp-list-container flex-column-container">
        <li class="add-user-button-container"></li>
      </ul>
    `;
```

별거 아닌거지만 인덴트를 맞춰주시면 코드를 보기 더 좋을거 같아요!

Reply:
평소 프리티어에 의존하다보니 템플릿 리터럴 같은 경우 종종 이렇게 인덴트를 맞추지 않고 넘기는 버릇이 생긴거 같습니다.
인덴트도 신경쓰도록 할게요..!